src/components/Dialog.tsx

```diff
@@ -111,7 +111,7 @@ export const Modal = styled.div<{ color: Color }>`
   position: absolute;
   top: 0;
   width: 100%;
-  z-index: ${Layer.DIALOG};
+  z-index: 1;
```

Comment:
there isn't a layer for 1, I can add one if wanted

Reply:
Why are you not using `Layer.DIALOG` for the `Dialog`?
If this is shared by another layer, then _that_ layer should be renamed.

Reply:
if you look at the description you can see the modal is overlapping on the navbar.  Changing this to a z-index of 1 fixes that.

Reply:
z-index's should only be done through the `Layer` enum to keep them organized.
This should be updated in src/theme/layer.ts, and continue using `Layer.DIALOG`.

Reply:
lynn solved this issue here: https://github.com/Uniswap/interface/pull/4576.  I'm going to revert